guides/v2.1/migration/migration-overview-how.md

```diff
@@ -1,52 +1,39 @@
 ---
 group: migration
-subgroup: A_Overview
 title: How migration works
-menu_title: How migration works
-menu_node:
-menu_order: 3
 version: 2.1
 github_link: migration/migration-overview-how.md
 redirect_from: /guides/v1.0/migration/migration-overview-how.html
-functional_areas:
-  - Tools
 ---
 
 ## Terminology
 
 This document uses the following terminology to discuss the Data Migration Tool:
 
-* **Step:** A unique migration task that must be executed in a prescribed order
-
-* **Map:** A set of rules that describe connections between Magento 1.x and Magento 2.x data structures
-
-* **Mode:** Represented by a separate Data Migration Tool command, defines the basic mode of operation as:
-
-  * *Settings:* Migrates the system configuration and website-related settings
-
-  * *Data:* Migrates database assets in bulk
-
-  * *Delta:* Migrates incremental changes (for example, orders and inventory)
+* **Step** - unique migration task that must be executed in a prescribed order
+* **Map** - set of rules that describe connections between Magento 1.x and Magento 2.x data structures
+* **Mode** - represented by a separate Data Migration Tool command; defines the basic mode of operation as:
+  * *Settings* - migrates the system configuration and website-related settings
+  * *Data* - migrates database assets in bulk
+  * *Delta* - migrates incremental changes, such as orders and inventory
 
 ## Migration modes (phases)
 
-This guide provides information about migration using the Data Migration Tool that can transfer and adapt data from Magento 1 to Magento 2 stores. The migration in overall is split into three phases (or "modes"):
+This guide provides information about migration using the Data Migration Tool that can transfer and adapt data from Magento 1 to Magento 2 stores. The migration in overall is split into three phases (or modes):
```

Comment:
Simplify and clarify. Maybe something like: "The data migration tool transfers and adapts data from Magento 1 to Magento 2 stores. The tool works by splitting the migration into three phases (or modes):"

Reply:
👍